1-exercises/A-reading-list/script.js

```diff
@@ -1,5 +1,21 @@
 function readingList(books) {
-  // Write your code here...
+  let html= document.createElement("ul");
+    
+    for(let x of books){
+      let className = x.alreadyRead ? 'green' : 'red';
+      let bookElement= document.createElement("li");
+      let paraElement= document.createElement("p");
+      let imgElement= document.createElement("img");
+      paraElement.innerText = `${x.title} by ${x.author}`;
+      imgElement.src =`${x.bookCoverImage}`;
+      bookElement.classList.add(className);
+      bookElement.appendChild(paraElement);
+      bookElement.appendChild(imgElement);
+      html.appendChild(bookElement);
+     
+    } 
```

Comment:
This is a very tidy solution. Logical - create your elements then build the subtree.